db/migrate/20160718222846_add_gender_to_profiles.rb

```diff
@@ -0,0 +1,5 @@
+class AddGenderToProfiles < ActiveRecord::Migration
+  def change
+    add_column :profiles, :gender, :string
```

Comment:
This choice of datatype is interesting. It gives you the most flexibility - but makes it harder to search on. At the very least I think there should be some validation to ensure that the given genders fall within a given list. Also consider enum, or even a separate model; could this be normalised?